src/main/frontend/format/mldoc.cljs

```diff
@@ -115,11 +115,15 @@
                        (seq macros)
                        (assoc :macros macros))
           alias (->vec-concat (:roam_alias properties) (:alias properties))
-          tags (->vec-concat (:roam_tags properties) (:tags properties) definition-tags)
+          filetags (if-let [org-file-tags (:filetags properties)]
+                     (->> (string/split org-file-tags ":")
+                          (remove string/blank?)))
+          tags (->vec-concat (:roam_tags properties) (:tags properties) definition-tags filetags)
```

Comment:
@tiensonqin So one thing I noticed while testing is that this would work when expected only when `#+ROAM_TAGS` has a single tag. When there are multiple (I believe they can be whitespace separated), you'd end up with `tags ("tag1 tag2 tag3")`, whereas you probably expect `tags ("tag1" "tag2" "tag3")`. I guess this code is recent https://github.com/logseq/logseq/issues/1031, do you want me to fix that as well? Or you'd prefer a separate PR?

Reply:
@Kamalnrf Indeed!
I just checked the org-roam documentation, it says:
> 'prop Extract tags from the #+roam_tags property. Tags are space delimited. Tags may contain spaces if they are double-quoted. e.g. #+roam_tags: TAG "tag with spaces"

Yes, please fix it in this PR, thanks a lot!

Reply:
done! somewhat messy, but works :shrug: